compiler/testData/codegen/box/ranges/unsigned/expression/overflowZeroDownToMaxValue.kt

```diff
@@ -1,4 +1,7 @@
 // KJS_WITH_FULL_RUNTIME
+// TODO: muted automatically, investigate should it be ran for JVM_IR or not
```

Comment:
Do you understand why these two tests are now failing. If so, can you replace this auto mute comment with an actual comment explaining what is going on?

Reply:
Will do. Mark was wondering the same thing earlier. I'll add a comment to the generator for these tests.